src/main/kotlin/extensions/wu/seal/ForceInitDefaultValueWithOriginJsonValueSupport.kt

```diff
@@ -1,6 +1,7 @@
 package extensions.wu.seal
 
 import extensions.Extension
+import groovy.json.StringEscapeUtils
```

Comment:
Please try not to use class in the groovy package as we also have a java library to generate kotlin data class.
If include groove, the java library also need to include it, then the library will be bigger 🤣